app/redux/selectors/SetsSelectors.js

```diff
@@ -377,7 +377,7 @@ export const hasChangesToSync = (state) => {
 
 const getBestEverOfMetric = (state, set, metricFunction, isMax=true) => {
     // null if not enough data entered
-    if (!isSetComparable(set)) {
+    if (!isSetComparable(set) || SetUtils.isDeleted(set)) {
```

Comment:
- [ ] this only checks if the set itself is deleted, but it doesn't check if the sets it is COMPARING against are deleted